content/rest-api/resources.md

```diff
@@ -1079,11 +1079,54 @@ Below is the JSON standard format representing a published product. Notice how t
 Endpoints for the published products are only available starting the 2.0 version.
 :::
 
-::: panel-link Want more details about the published product resource? [Check its endpoints here!](/api-reference.html#Publishedproducts)
+::: panel-link Want more details about the published product resource? [Check its endpoints here!](/api-reference.html#Publishedproduct)
 :::
 
 ## Reference entities
 
+### Reference entity (3.x and EE only)
+
+Reference entities have been introduced in the PIM in the 3.0 version.
+
+In the Akeneo UI, you can find the reference entities in the `Entities` menu. Below is an example of a reference entity in the UI.
+
+![Reference entity](/img/screenshots/...png)
+
+Below is the JSON standard format representing a reference entity.
+
+```json
+{
+  "code": "brands",
+  "attributes": {
+    "description": {
+      "labels": {
+        "en_US": "Description",
+        "fr_FR": "Description"
+      },
+      "type": "",
```

Comment:
"type": "text" ? (same for "country")

Reply:
My bad. The attributes are not embedded anymore in the ref entity. They will have their own route.